### PR TITLE
Fix app update notifications not being scheduled

### PR DESCRIPTION
**Pull request by janhavisinghh**

### CODE REVIEW

**Comment by janhavisinghh** [comment]
Testing this in multiple devices. Will open for review after that's done.

File: app/src/main/java/org/simple/clinic/appupdate/AppUpdateNotificationWorker.kt [hunk 1]
```diff
@@ -51,13 +50,14 @@ class AppUpdateNotificationWorker(
     private const val NOTIFICATION_CRITICAL = 6
     private const val NOTIFICATION_CHANNEL_NAME = "Updates"
 
-    fun createWorkRequest(userClock: UserClock, schedule: AppUpdateNotificationSchedule): PeriodicWorkRequest {
+    fun createWorkRequest(userClock: UserClock, schedule: AppUpdateNotificationSchedule): OneTimeWorkRequest {
```
Comment (msasikanth) [inline app/src/main/java/org/simple/clinic/appupdate/AppUpdateNotificationWorker.kt:53]: It looks like we are using `OneTimeWorkRequest` but are not rescheduling it once it's done.

**Review by msasikanth — CHANGES_REQUESTED** [review]

File: app/src/main/java/org/simple/clinic/appupdate/AppUpdateNotificationWorker.kt (hunk 1 above)
Comment (janhavisinghh) [reply app/src/main/java/org/simple/clinic/appupdate/AppUpdateNotificationWorker.kt:53]: oh, good catch.

**Comment by janhavisinghh** [comment]
@msasikanth 
Testing steps: 
- Download sbx apk from [https://app.bitrise.io/build/95b2a5c8-e944-4d78-85d8-e081deb37a7d](https://app.bitrise.io/build/95b2a5c8-e944-4d78-85d8-e081deb37a7d)
- Add latest sbx build number to firebase remote config
- Test if the notifications are working as expected

**Review by msasikanth — APPROVED** [review]

**Pull request merged**